packages/errors/tsconfig.json

```diff
@@ -2,7 +2,6 @@
   "extends": "../ts/tsconfig.json",
   "include": [
     "src",
-    "test",
```

Comment:
don't the tests need type-checking? if they require a different config, you could add it in `packages/error/test/tsconfig.json`

Reply:
They do need type checking. 

Tests are ignored in most other packages. Therefore, I removed them from this package as well. 

I wanted this PR to stay small, so I removed it here too.

Reply:
This seems like a regression in test/type coverage if you had to remove type checking from `@packages/errors/test`. I don't follow the reasoning of "I wanted this PR to stay small, so I removed it here too.", I don't think.